### PR TITLE
Tutorial on how to make wallet transactions on the web using aepp-sdk-js

### PR DESCRIPTION
**Pull request by Jesulonimi21**
Created new tutorial

### CODE REVIEW

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    return sdkInstance;
+}
+```
+To create an SDK instance, we need to specify a Node Url `NODE_URL` which states if to connect to the mainnet or the testnet, Connecting to the testnet is enough for this tutorial so we give it a value of https://sdk-testnet.aepps.com, then we create a memory account which uses the secret key and public key our function receives to do this. Then we further create our `nodeInstance` using the `NODE` constructor of the global Ae object. Lastly, we create an SDK instance using the `Universal` constructor on the global `Ae` object and return it. Let's proceed by finally adding the function that will allow us to check our Aeternity wallet account balance.
+
+```javascript
+  async function getAccountBalance(publicKey,sdkInstance){
+    let height=await sdkInstance.height();
+    try{
+        let balance=await sdkInstance.balance(publicKey,{height:+height,hash:null});
```
Comment (thepiwo) [inline making-aeternity-wallet-transactions-with-aepp-sdk-js.md:97]: `getBalance` defaults to 0 if account doesn't exist.

no need to pass `height, hash` if not used or default values

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        console.log(balance);
+        balance=parseFloat(balance)/1000000000000000000;
```
Comment (thepiwo) [inline making-aeternity-wallet-transactions-with-aepp-sdk-js.md:99]: this yield rounding errors in JS eventually, use a BigNumber library

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      const publicKey=Ae.Crypto.aeEncodeKey(keyPair.publicKey);
+      const secretKey=buffer.Buffer.from(keyPair.secretKey).toString('hex')
+      return{publicKey:publicKey,privateKey:secretKey};
+    }
+ ```
+ What we do above is pretty simple, the function above expects a private key as a string and then converts it to an ArrayBuffer using the `hexStringToByte` function in the crypto module of the aepp-sdk-js. We further access the `generateKeyPairFromSecret` function in the crypto module of the aeternity javascript SDK, this function then returns a KeyPair object containing both the secret key and private key as an ArrayBuffer of view UInt8Array. The next step is to retrieve the private key and public key from their respective ArrayBuffers, we do this using the `aencodeKey` function of the `Crypto` module and the `from` method of the Buffer module to get the private key and public Key as strings. So now that you know how to get a public key from an existing private key. let's go over creating a KeyPair from a brand new public key and private key.
+ 
+ ### New KeyPair Creation Without Existing Public Key
+```javascript
+ function generateKeyPairWithoutSecretKey(){
```
Comment (thepiwo) [inline making-aeternity-wallet-transactions-with-aepp-sdk-js.md:64]: I think its better to connect to existing wallet, keypairs shouldn't be dealt with manually

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md (hunk 1 above)
Comment (Jesulonimi21) [reply making-aeternity-wallet-transactions-with-aepp-sdk-js.md:97]: Noted, i corrected that in the showcase and the tutorial video already, I forgot to correct it here

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md (hunk 2 above)
Comment (Jesulonimi21) [reply making-aeternity-wallet-transactions-with-aepp-sdk-js.md:99]: ok, noted

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md (hunk 3 above)
Comment (Jesulonimi21) [reply making-aeternity-wallet-transactions-with-aepp-sdk-js.md:64]: Yeah, there is  a function that i created and explained in the tutorial that also shows how to do that

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md (hunk 2 above)
Comment (Jesulonimi21) [reply making-aeternity-wallet-transactions-with-aepp-sdk-js.md]: Have corrected this now

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md (hunk 1 above)
Comment (Jesulonimi21) [reply making-aeternity-wallet-transactions-with-aepp-sdk-js.md]: Have corrected this now

File: making-aeternity-wallet-transactions-with-aepp-sdk-js.md (hunk 3 above)
Comment (Jesulonimi21) [reply making-aeternity-wallet-transactions-with-aepp-sdk-js.md]: So have deleted the part that explains how to do this and left the part that explains how to connect to an existing wallet using its private key

**Comment by omar-saadoun** [comment]
@thepiwo thanks for reviewing this, it is ok now to merge?

**Pull request merged**